SAMtemplates/lambda_resources.yaml

```diff
@@ -201,10 +212,7 @@ Resources:
     Properties:
       CodeUri: ../packages/
       Handler: splunkProcessor.handler
-      Runtime: nodejs18.x
       Role: !GetAtt SplunkDeliveryStreamProcessorRole.Arn
-      Architectures:
-        - x86_64
       MemorySize: 512
       Timeout: 60
       Environment:
```

Comment:
Should the NODE_OPTIONS be removed from here too?

Reply:
yep - removed it here